core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -327,6 +327,25 @@ public T keepAliveWithoutCalls(boolean enable) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Sets the retry buffer size in bytes. If the buffer limit is exceeded, no RPC
```

Comment:
We need some documentation saying these do little today, since retry is not enabled.